### DISCUSSION TITLE
checking for non-Latin characters in the selection

### DISCUSSION THREAD

**Discussion by shtoormanSam** [dbeaver/dbeaver#19551] [Ideas]
It happens that in the variable name there is a non-Latin character that looks exactly like Latin, but has a different code. Such errors (typos) are very difficult to catch visually.

It would be nice to have a hotkey to check the selection for non-Latin characters and highlight them.

**Comment by ShadelessFox** [comment]
Hello @shtoormanSam,

Thank you for your request.

Do you aware of any other tools that have a similar feature?

**Comment by shtoormanSam** [comment]
No, you can get there first ))

ср, 15 мар. 2023 г. в 20:21, ShadelessFox ***@***.***>:

**Comment by LonwoLonwo** [comment]
Hello @shtoormanSam 

DBeaver PRO has a very similar feature from the last release - a spelling service. It can indicate non-Latin characters in your text in SQL Editor. And in the Data Editor in the future.

You can try it for free with our trial https://dbeaver.com/trial/ on the Early Access version https://dbeaver.com/files/ea/
